ext/thread_test/environments/production/modules/threader/manifests/params_class.pp

```diff
@@ -0,0 +1,3 @@
+class threader::params_class (String $input = 'softserve') {
```

Comment:
Why does this have a default value but the other one doesn't?

Reply:
Oh, I was testing something. But I don't mind having them be different, actually. We prefer the data in hiera in both cases, maybe I should remove the hiera data from the one with the default to test that failed lookup and falling through behavior 🤔

Reply:
yeah, that's what I thought you were going for at first